__tests__/app.test.js

```diff
@@ -74,3 +75,44 @@ describe("TASK 4 --- GET - /api/articles/:article_id", () => {
     })
   });
 });
+describe('TASK 5 --- GET - /api/articles', () => {
+  it('GET - status: 200 - returns array of articles that have the correct shape and data-types', () => {
+    return request(app)
+    .get(`/api/articles`)
+    .expect(200)
+    .then((response) => {
+      expect(response.body.articles.length).toBe(12)
+      response.body.articles.forEach((item) => {
+        expect(typeof item.article_id).toBe("number")
+        expect(typeof item.title).toBe("string")
+        expect(typeof item.topic).toBe("string")
+        expect(typeof item.author).toBe("string")
+        expect(typeof item.created_at).toBe("string")
+        expect(typeof item.votes).toBe("number")
+        expect(typeof item.comment_count).toBe("string")
+      })
+    })
+  });
+  it('GET - status: 200 - returns correctly counted comments for a given article', () => {
+    return request(app)
+    .get(`/api/articles`)
+    .expect(200)
+    .then((response) => {
```

Comment:
We want to also check the length of the data when doing a forEach - to ensure we get all the expected data back